test/runtests.jl

```diff
@@ -22,6 +22,7 @@ end
         for i in 1:length(evt.waveforms)
             signalsum += abs(evt.waveforms[i].value[end])
         end
+        @info signalsum
```

Comment:
Do we want a more informative `@info` than just `signalsum` as the number?

Reply:
I just added this line such that I see the number when I locally tests part of the tests.

Reply:
Will we leave this for the tests on GitHub as well?